pontoon/insights/static/js/insights_tab.js

```diff
@@ -0,0 +1,953 @@
+                backgroundColor: gradient,
+                borderColor: ['#7BC876'],
+                borderWidth: 2,
+                pointBackgroundColor: '#7BC876',
+                pointHitRadius: 10,
+                pointRadius: 4,
+                pointHoverRadius: 6,
+                pointHoverBackgroundColor: '#7BC876',
+                pointHoverBorderColor: '#FFF',
+              },
+              humanData.length > 0 && {
+                type: 'bar',
+                label: 'Human translations',
+                data: humanData,
+                yAxisID: 'strings-y-axis',
+                backgroundColor: '#4f7256',
+                hoverBackgroundColor: '#4f7256',
+                stack: 'translations',
+                order: 2,
+              },
+              machineryData.length > 0 && {
+                type: 'bar',
+                label: 'Machinery translations',
+                data: machineryData,
+                yAxisID: 'strings-y-axis',
+                backgroundColor: '#41554c',
+                hoverBackgroundColor: '#41554c',
+                stack: 'translations',
+                order: 1,
+              },
+              newSourcesData.length > 0 && {
+                type: 'bar',
+                label: 'New source strings',
+                data: newSourcesData,
+                yAxisID: 'strings-y-axis',
+                backgroundColor: '#272a2f',
+                hoverBackgroundColor: '#272a2f',
+                stack: 'source-strings',
+                order: 3,
+                hidden: true,
+              },
+            ].filter(Boolean), // Filter out empty values
+          },
+          options: {
+            legend: {
+              display: false,
+            },
+            legendCallback: Pontoon.insights.customLegend(chart),
+            tooltips: {
+              mode: 'index',
+              intersect: false,
+              borderColor: '#7BC876',
+              borderWidth: 1,
+              caretPadding: 5,
+              xPadding: 10,
+              yPadding: 10,
+              itemSort: function (a, b) {
+                // Dataset order affects stacking, tooltip and
+                // legend, but it doesn't work intuitively, so
+                // we need to manually sort tooltip items.
+                if (a.datasetIndex === 2 && b.datasetIndex === 1) {
+                  return 1;
+                }
+              },
+              callbacks: {
+                label: function (items, chart) {
+                  const human = chart.datasets[1].data[items.index];
+                  const machinery = chart.datasets[2].data[items.index];
+
+                  const label = chart.datasets[items.datasetIndex].label;
+                  const value = items.yLabel;
+                  const base = label + ': ' + nf.format(value);
+
+                  switch (label) {
+                    case 'Completion':
+                      return base + '%';
```

Comment:
I don't remember the history here, but there is no need manually format percentages like this; that's why there's the `style: 'percent'` option on `Intl.NumberFormat`.